src/Vimeo.js

```diff
@@ -67,8 +73,7 @@ THE SOFTWARE. */
       this.iframe.setAttribute('allowFullScreen', '0');
 
       var divWrapper = document.createElement('div');
-      divWrapper.setAttribute('style', 'margin:0 auto;padding-bottom:56.25%;width:100%;height:0;position:relative;overflow:hidden;');
-      divWrapper.setAttribute('class', 'vimeoFrame');
+      divWrapper.setAttribute('style', 'width:638px; height:358px; overflow:hidden; margin:0 auto;');
```

Comment:
Where are these heights and widths coming from? They seem to be giving me a vimeo video with a grey border around it (Not proper sizing).